lib/utils.pm

```diff
@@ -2486,7 +2486,8 @@ sub install_patterns {
         # skip installation of wsl_base, wsl_gui and wsl_systemd patterns due to bsc#1226314.
         next if (($pt =~ /wsl_base|wsl_gui|wsl_systemd/) && check_var('PATTERNS', 'all'));
         # if pattern is common-criteria and PATTERNS is all, skip, poo#73645
-        next if (($pt =~ /common-criteria/) && check_var('PATTERNS', 'all'));
+        # if pattern is fips-certified and PATTERNS is all, skip, bsc#1228852
+        next if (($pt =~ /common-criteria|fips-certified/) && check_var('PATTERNS', 'all'));
```

Comment:
I think need to add "fips" here, too, otherwise it would install "fips" pattern here: https://openqa.suse.de/tests/15149134#step/patch_sle/113

Reply:
changed and rerun the VR.